en/sensor/lidar_lite.md

```diff
@@ -38,18 +42,22 @@ Lidar-Lite v2 and v3 are the same, except that the order of pins in the connecto
 
 ### PWM Interface Wiring
 
-The pin connections for wiring LidarLite to the *Pixhawk 1* AUX ports (PWM interface) are shown below. 
+The pin connections for wiring LidarLite to the *Pixhawk 1* AUX ports (PWM interface) are shown below.
 
 Pin | Lidar-Lite (v2, v3) | Pixhawk AUX Servo | Comment
 --- | ---   | --- | ---
 1   | VCC   | AUX 6 (center) | Power supply. 4.75-5.5V DC Nominal, Maximum 6V DC.
 2   | RESET | AUX 6 (bottom) | Reset line of the sensor
-3   | PWM   | AUX 5 (bottom) | PWM input of the Lidar Lite. **Needs a 470 Ohm pull-down (to GND), Do not use a 1 K0hm resistor.**
+3   | PWM   | AUX 5 (bottom) | PWM output of the Lidar Lite. **Needs a 470 Ohm pull-down (to GND), Do not use a 1 K0hm resistor.**
 4   | SCL   | - | Not connected
 5   | SDA   | - | Not connected
-6   | GND   | AUX 6 (top)    | Ground
+6   | GND   | AUX 6 (top) | Ground
 
-The wiring for LidarLite v2 is shown below. Lidar-Lite v3 is wired similarly, except that the pin-numbering on the connector is reversed.
+> **Note** On a flight controller that has no AUX port the equivalent MAIN pins are used (e.g. the PWM output on the lidar instead maps to MAIN 5).
```

Comment:
@bkueng This captures your note on what happens on boards with no IO. I'm going to merge, but please sanity check this when you get a chance.

Reply:
Yup, should be good.

Reply:
Thanks!